src/GlobalStyle.js

```diff
@@ -8,21 +8,24 @@ const GlobalStyle = createGlobalStyle`
     font-family: Montserrat, sans-serif;
     font-weight: 700;
     font-size: 16px;
-    overflow-y: scroll;
   }
 
   html,
   body {
-    min-height: 100vh;
+    height: 100%;
+    width: 100%;
   }
 
   .page {
     display: flex;
     flex-direction: column;
-    justify-content: center;
+    justify-content: flex-start;
     align-items: center;
     flex-grow: 1;
-    padding-top: 22px;
+    position: relative;
+    min-height: 100vh;
```

Comment:
Having a min-height is a good idea. But the complete page should have min-height 100vh, so here it should be 100vh - headerHeight - footerHeight.

I guess it would be good if you create a [theme](https://styled-components.com/docs/advanced#theming) where you can store these heights, colors and so on